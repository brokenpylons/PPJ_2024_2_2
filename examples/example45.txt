((a * b) / (c + d))^0.75 + 4 * e^(2.2 * f) / (g - 1)
