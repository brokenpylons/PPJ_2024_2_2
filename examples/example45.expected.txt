lparen("(") lparen("(") variable("a") times("*") variable("b") rparen(")") divides("/") lparen("(") variable("c") plus("+") variable("d") rparen(")") rparen(")") pow("^") real("0.75") plus("+") real("4") times("*") variable("e") pow("^") lparen("(") real("2.2") times("*") variable("f") rparen(")") divides("/") lparen("(") variable("g") minus("-") real("1") rparen(")") 
